src/utils.py

```diff
@@ -4186,11 +4186,14 @@ def commit(self, overlay: bool = True) -> None:
         self.totalcont = ""  # re-use
 
 
-def apply_redactions(page: fitz.Page, images: int = 2, graphics: int = 1) -> bool:
+def apply_redactions(page: fitz.Page, keep_text: int = 0, images: int = 2, graphics: int = 1) -> bool:
```

Comment:
Could we append the new `keep_text` arg to the fns args? Otherwise existing code that specifies things as positional args will break. [We should probably move to using `*` to force naming of args like these.]

The new `keep_text` arg seems to behave in the opposite way to the existing args, which could be confusing. Could we instead call it `text` and use zero to mean "ignore text" and one to mean "remove overlapping text"?

Reply:
I will change this to using the `*` parameter - good point.
Otherwise, the reason for choosing the name `keep_text` is two-fold:
1. When interpreting 0 as False (as is usual), the condition could sound like "text is false", giving rise to the wrong association text will not be removed.
2. The original intent of redactions is to remove text - for legal / data protection reasons. From this perspective, **_not_** removing text is a paradox desire, which can justify an uncommon argument naming.